.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       max-parallel: 15
       matrix:
-        node: [14.x, 16.x, 18.x]
+        node: [16.x, 18.x]
         python: ["3.7", "3.9", "3.11"]
```

Comment:
Python 3.7 is also EOL this month...  What about just `python: [3.8, 3.11]`?
* https://devguide.python.org/versions/